### ISSUE TITLE
training loss does not decrease

### ISSUE THREAD

**Issue by CZ-Wu** [facebookresearch/DeepSDF#56]
I train the network in the airplane category, but the training loss does not decrease after about 40 epoch. I got the same error as mentioned in [https://github.com/facebookresearch/DeepSDF/issues/31#issue-506755157](url) during reconstruction. The output sdf values are between -0.0005 and 0.0005  during training, but the output sdf values are all positive or all negative when I try to reconstruct the shape from latent code. Here is the curve of training loss.
![image](https://user-images.githubusercontent.com/27475044/94093382-a27a0a80-fe4f-11ea-841c-4aca8d48fbb2.png)

**Comment by ChushanZhang** [comment]
I met the same issue, and the loss stopped decreasing after around 176 epochs.

The LR I set is 1e-5 * batch size where the scheduler is (beta = 100, factor = 0.5).

I hope someone can guide me on what was going on.
